docs/modules/ROOT/pages/get-started.adoc

```diff
@@ -1,9 +1,246 @@
-= Get Started with Hazelcast CLC
+= Get Started With the Hazelcast CLC
+:description: In this tutorial, you'll learn the basics of how to configure, start up, and connect the Hazelcast Command Line Client (CLC) to a {hazelcast-cloud} cluster. You'll also see how to switch between clusters using the CLC. Finally, you'll perform some basic operations on a cluster from the command line and then by running a script to automate the same actions.
```

Comment:
```suggestion
:description: In this tutorial, you'll learn the basics of how to configure the Hazelcast Command Line Client (CLC), start it, and connect it to a {hazelcast-cloud} cluster. You'll also see how to switch between clusters, using the Hazelcast CLC. Finally, you'll perform some basic operations on a cluster from the command line and then by running a script to automate the same actions.
```

The comma before 'using' makes it clear that it's the user who does the action not the clusters.

Reply:
Updated as requested.